pages/facts.html

```diff
@@ -4,9 +4,90 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Facts About Tacos</title>
+  <link href="../styles/style.css" rel="stylesheet">
+  <link href="../styles/facts.css" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Permanent+Marker&ampdisplay=swap" rel="stylesheet"> 
 </head>
-<body>
-  
-</body>
+  <body>
+    <div class="container">
+      <div class="header">
+        <header>
+          <h1 id="fancyfont">
+            Facts About Tacos
+          </h1>
+          <nav>
+            <ul>
+              <li>
+                <a href="index.html">Home</a>
```

Comment:
It's a little subjective, but in this layout I might consider the header and the section as part of the significant content of the page and place it inside a `main` tag.